microsoft-edge/webview2/concepts/versioning.md

```diff
@@ -3,15 +3,22 @@ description: Versioning Models used for Microsoft Edge WebView2
 title: Versioning of Microsoft Edge WebView2
 author: MSEdgeTeam
 ms.author: msedgedevrel
-ms.date: 10/19/2020
+ms.date: 10/28/2020
 ms.topic: conceptual
 ms.prod: microsoft-edge
 ms.technology: webview
 keywords: IWebView2, IWebView2WebView, webview2, webview, wpf apps, wpf, edge, ICoreWebView2, ICoreWebView2Host, browser control, edge html
 ---
 # Understand WebView2 SDK versions  
 
-To develop a WebView2 application, you must install either the [WebView2 Runtime][MicrosoftDeveloperEdgeWebview2] or a [non-stable Microsoft Edge channel][MicrosoftedgeinsiderDownload].  The minimum version that's required is included in the NuGet package version of the SDK.  For example, if you use the `SDK package version 0.9.488`, then you must install either the [WebView2 Runtime][MicrosoftDeveloperEdgeWebview2] or a [non-stable Microsoft Edge channel][MicrosoftedgeinsiderDownload] with a build number of 488 or later.  The minimum version required is also specified in the WebView2 [Release Notes][Releasenotes].  New versions of the WebView2 SDK are shipped at the same general cadence as the Microsoft Edge \(Chromium\) browser, which is approximately every six weeks.  
+To develop a WebView2 application, you must install either the [WebView2 Runtime][MicrosoftDeveloperEdgeWebview2] or a [non-stable Microsoft Edge channel][MicrosoftedgeinsiderDownload].  To determine the minimum version that's required chose the most recent of:
```

Comment:
The most recent between those two will always be the min version required by interface.

I might say like 'The minimum runtime version required by an SDK is listed in the release notes. The SDK will work with any runtime version greater than that. But new interfaces and APIs are added in later versions. If your app depends on an API added in a later version then your app may depend on a version of the runtime greater than the SDK's minimum version.'